bin/crew

```diff
@@ -104,9 +105,9 @@ end
 
 String.use_color = args["--color"] || !args["--no-color"]
 @opt_keep = args["--keep"]
-@opt_verbose = args["--verbose"]
+Verbose = args["--verbose"]
```

Comment:
Lowercase `v`?

Reply:
This is a constant

Reply:
I get that but this change is not consistent with the other options.  If we are going to make this kind of a change, we should make it the same for all the other arguments too.  Ex: `@opt_keep`

Reply:
The changes of other arguments can be added in further refactoring

Reply:
Is it not ruby style to use (lowercase) `snake_case` for symbols, methods, and variables?

Reply:
I don't think there is a standard for variable names? (like the `pkgName` variable in `bin/crew`)

Reply:
I don't believe variables are capitalized because they can get confused with text (like words from a sentence).  Environment variables are generally all caps but other variables start with lowercase as @satmandu mentioned.  camelCase is acceptable too, although, I'm old school and prefer underscores for clarity.